threadsd/main.go

```diff
@@ -118,6 +119,7 @@ func main() {
 			return true
 		}),
 		grpcweb.WithWebsockets(true),
+		grpcweb.WithWebsocketPingInterval(*keepAliveInterval),
```

Comment:
This is the same thing that the improbable-eng lib does. They pass along keepAliveInterval regardless of value, and if it is less than 1 second, it is ignored.